src/components/OSCALLoader.js

```diff
@@ -216,10 +217,7 @@ export default function OSCALLoader(props) {
       setIsResolutionComplete(true);
       setHasDefaultUrl(false);
     } else {
-      setOscalUrl(props.oscalObjectType.defaultUrl);
-      loadOscalData(props.oscalObjectType.defaultUrl);
-      setIsLoaded(false);
-      setIsResolutionComplete(false);
+      setOscalUrl(props.oscalUrl || props.oscalObjectType.defaultUrl);
```

Comment:
I think this bit is what broke things. @tuckerzp I am not going to have time for a little bit today to work on this. I think if this matches what we do on line 62-63 it should go back to working?

Reply:
I will look into this.